### ISSUE TITLE
Multitask Pre-training

### ISSUE THREAD

**Issue by MichaelFu1998-create** [salesforce/CodeT5#51]
Dear authors of CodeT5,
Thanks for contributing such an amazing model to the community.
In the paper, it is said that CodeT5 was pre-trained using multiple tasks.
I'm wondering how these task were arranged, did you pre-train multiple tasks all-in-one and combine the loss or did you pre-train each task one by one.
Thank you very much for your help :)

Kind regards
Michael

**Comment by yuewang-cuhk** [comment]
Hi, we sample one task for each batch based on a balanced sampling strategy during multi-task pretraining. Please refer to [here](https://github.com/salesforce/CodeT5/blob/5b37c34f4bbbfcfd972c24a9dd1f45716568ecb5/run_multi_gen.py#L282) for more details.
